frontend/src/views/Nodes/components/TNodesLogsContentList.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="logs__list">
+    <div class="logs__list-heading">
+      <div class="logs__list-heading-wrapper">
+        <p v-if="isDateColVisible" class="logs__list-heading-name">Date</p>
+        <p v-if="isTimeColVisible" class="logs__list-heading-name">Time</p>
+        <p class="logs__list-heading-name">Message</p>
+      </div>
+      <div class="logs__list-checkbox">
+        <t-checkbox
+          label="Follow Logs"
+          :checked="follow"
+          @click="() => (follow = !follow)"
+        />
+      </div>
+    </div>
+    <ul
+      class="logs__view"
+      :class="{ 'logs__view--short': $route.params.service === 'dmesg' }"
+      ref="logView"
+    >
+      <li class="logs__item" v-for="(log, index) in filteredLogs" :key="index">
+        <div v-if="isDateColVisible" class="logs__item-date">
+          {{ log.date }}
+        </div>
+        <div v-if="isTimeColVisible" class="logs__item-time">
+          {{ log.time }}
+        </div>
+        <div class="logs__item-message">
+          {{ log.message }}
+        </div>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
Not sure if that's for the next PRs but would be nice to hightlight the log part which was found by search.